AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -912,6 +912,7 @@ video.foxbusiness.com##.notification-banner
 braintreepayments.com###gdprComponent
 magzter.com##.cookies__notification
 istikbal.com.tr,bellona.com.tr,mondi.com.tr##.cookiepopup-container
+dreamstage.live##.CookiePopUp_container__17RPg
```

Comment:
You can improve your rule `dreamstage.live##div[class^="CookiePopUp_"]`. Also we have a similar one.